main/webapp/modules/core/langs/translation-zh.json

```diff
@@ -128,6 +128,7 @@
     "core-dialogs/no-cluster-found": "选中的操作并没有生成簇",
     "core-dialogs/try-another-method": "请尝试上面的其他操作或者修改操作的参数",
     "core-dialogs/clustering": "簇集中…",
+    "core-dialogs/sqlExportEscapeCharacter": "转义字符",
```

Comment:
Here also, there is naming convention to follow. No uppercase, and dash to separate words.
```suggestion
    "core-dialogs/sql-export-escape-char": "转义字符",
```

What do you think of that?

Regards, Antoine

Reply:
I think my way is better because I follow the other three labels.
![image.png](https://s2.loli.net/2022/04/24/siZM1X2WV8rLQOY.png)

Reply:
> I think my way is better because I follow the other three labels. ![image.png](https://camo.githubusercontent.com/9371dc24714d33ca6d4d4168677dbfb507aa3185cadda421445821f7915e55d1/68747470733a2f2f73322e6c6f6c692e6e65742f323032322f30342f32342f73695a4d315832575638724c514f592e706e67)

Wow @the-star-sea! You are making an intersting observation. But the sad thruth is probably that you've unearthed what are mistakes. There are a few other cases, but it's rare.
"core-index/rowCount", "core-index/customMetadata", "core-index/id": "项目编号：", "core-index/importOptionMetadata"

@wetneb: dou we have a rule for the naming of the translation strings?

Regards, Antoine